code/game/objects/items/implants/implant_hijack.dm

```diff
@@ -69,3 +79,17 @@
 		return TRUE
 	object.attack_ai(imp_in)
 	return TRUE
+
+/* for later
+/obj/item/implant/hijack/proc/hijack_remotely(/obj/machinery/power/apc/apc)
```

Comment:
If it's a WIP, keep it off the repository and don't clog up this PR with commented features.

Reply:
sure, I'll pop it into another branch